src/llm/http_llm_calculator.cc

```diff
@@ -113,6 +121,16 @@ class OpenAIChatCompletionsRequest {
     }
 
     chat_t getMessages() const { return this->messages; }
+    Endpoint getEndpoint() const { return this->endpoint; }
+    void setEndpoint(Endpoint endpoint) { this->endpoint = endpoint; }
+    std::string getPrompt() const {
+        if (this->prompt.has_value()) {
+            return this->prompt.value();
+        } else {
+            return "";
+        }
+    }
```

Comment:
I suppose we can use value_or instead of this entire method: https://en.cppreference.com/w/cpp/utility/optional/value_or

Reply:
fixed